executor/compiler.go

```diff
@@ -125,6 +126,120 @@ func CountStmtNode(stmtNode ast.StmtNode, inRestrictedSQL bool) {
 	metrics.StmtNodeCounter.WithLabelValues(GetStmtLabel(stmtNode)).Inc()
 }
 
+// DbCountStmtNode records the number of statements with the same type and db.
+func DbCountStmtNode(stmtNode ast.StmtNode, inRestrictedSQL bool) {
```

Comment:
I think it is better to refactor `CountStmtNode` instead of writing a new function.

Reply:
ok